easybuild/easyconfigs/c/cpio/cpio-2.14-foss-2022a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
```

Comment:
this file needs to be removed

Reply:
removing the easyblock will result in :  No software-specific easyblock 'EB_cpio'

Reply:
what i meant is to remove the whole file, because now you have 2 easyconfigs in the PR.
you can do it from the cmd line as follows (note the colon in front):
```
eb --update-pr 19017 :cpio-2.14-foss-2022a.eb --pr-commit-msg "remove easyconfig"
```